setup.cfg

```diff
@@ -10,3 +10,10 @@ exclude = .venv,.git,.tox,docs,www_static,venv,bin,lib,deps,build
 
 [pydocstyle]
 match_dir = ^((?!\.|www_static).)*$
+
+[isort]
```

Comment:
yes, I added more information about it in this PR: https://github.com/home-assistant/home-assistant/pull/4401

I will remove it from this one.